code/game/machinery/electric.dm

```diff
@@ -0,0 +1,145 @@
+#define STASIS_TOGGLE_COOLDOWN 50
+/obj/machinery/electric
```

Comment:
Why is the type called just electric? I'd make more sense to be called electric_bed or something like that.

Reply:
well it started off as a subtype of static bed, but i later changed it